README.md

```diff
@@ -196,6 +217,36 @@ the URLs of different VOSK instances in JSON format:
 # org.jitsi.jigasi.transcription.vosk.websocket_url={"en": "ws://localhost:2700", "fr": "ws://localhost:2710"}
 ```
 
+Whisper configuration for transcription
+==================
+
+If you plan to use our own flavor of Whisper (check [jitsi/skynet](github.com/jitsi/skynet)), start by configuring the 
+following properties in `~/jigasi/jigasi-home/sip-communicator.properties`:
+
+```
+org.jitsi.jigasi.transcription.customService=EGHT_WHISPER
+org.jitsi.jigasi.transcription.whisper.websocket_url=wss://<YOUR-DOMAIN>:<<PORT>>
+```
+
+If you also plan to enable the ASAP authentication, have a look at the 
+[documentation](https://github.com/jitsi/skynet/blob/master/docs/streaming_whisper_module.md) and at the properties 
+in the transcription options section of this README.
+
+
+Oracle Cloud AI Speech configuration for transcription
+==================
+
+To use [Oracle Cloud AI Speech](https://docs.oracle.com/en-us/iaas/Content/speech/home.htm), you need to configure the 
+following properties in `~/jigasi/jigasi-home/sip-communicator.properties`:
+
+```
+org.jitsi.jigasi.transcription.customService=ORACLE_CLOUD_AI_SPEECH
+org.jitsi.jigasi.transcription.oci.websocketUrl=wss://realtime.aiservice-<<ENV>>.<<REGION>>.oci.oraclecloud.com
+```
+
+You also need to place valid OCI credentials under `~/.oci`. 
```

Comment:
Hum, by default jigasi runs as user jigasi and group jitsi and there is no home folder created for it, so that will not work with the debian packages. Isn't there another way to configure that?